2019-11-09 23:39:21,630	INFO	main1	<module>	Line:11	app started
2019-11-09 23:39:21,638	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-11-09 23:39:21,660	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-11-09 23:39:21,670	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-11-09 23:39:34,996	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:39:34] "GET /ProblemSet/addscore/2de84179-ecbf-491d-b924-da4f2053c1fe/106/0/1/fe0e8c652f8bfa912aed532b604f15e2 HTTP/1.1" 200 -
2019-11-09 23:39:35,000	INFO	_internal	_log	Line:122	192.168.2.210 - - [09/Nov/2019 23:39:34] "GET /ProblemSet/answer/2de84179-ecbf-491d-b924-da4f2053c1fe/2 HTTP/1.1" 200 -
2019-11-09 23:40:05,930	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
